baseplate/experiments/providers/variant_sets/multi_variant_set.py

```diff
@@ -0,0 +1,52 @@
+from .base import VariantSet
+
+
+class MultiVariantSet(VariantSet):
```

Comment:
Can you add docstrings for each of these `VariantSet`s?

Reply:
done